packages/runtime/src/context.ts

```diff
@@ -22,6 +22,8 @@ export interface RuntimeContext<Environment extends RuntimeEnvironment = Integra
      * Authenticated client to the GitBook API.
      */
     api: GitBookAPI;
+
+    waitUntil: FetchEvent['waitUntil'];
```

Comment:
```suggestion
    /**
     * Wait for any pending promises to complete before finishing the execution.
     * Use this function for any side-effects in your handlers that are not awaited, but must finish before
     * the execution context is closed.
     */
    waitUntil: FetchEvent['waitUntil'];
```

As this is a public API we should document it